### PR TITLE
[Form] Fixed overridden choices option in extended choice types

### PR DESCRIPTION
**Pull request by HeahDude**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #20771
| License       | MIT
| Doc PR        | ~

### CODE REVIEW

File: src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\Type;
+
+use Symfony\Component\Form\Forms;
+use Symfony\Component\Form\Tests\Fixtures\ChoiceTypeExtension;
+
+class ExtendedChoiceTypeTest extends \PHPUnit_Framework_TestCase
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php]: legacy TestCase, should use namespaced version

Comment (HeahDude) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php]: This in not a legacy test :). At least for now.

Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php]: the "underscore" is legacy in `PHPUnit_Framework_TestCase`, not the test case :)

**Review by xabbuh — APPROVED** [review]
👍 after the base test case class has been fixed

File: src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php (hunk 1 above)
Comment (HeahDude) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php]: Fixed, thanks!

**Comment by HeahDude** [comment]
Rebased, ready to merge!

**Comment by fabpot** [comment]
Thank you @HeahDude.

**Pull request merged**